modules/olm-bundle/01_mod.mk

```diff
@@ -53,6 +53,9 @@ olm-bundle: $(helm_chart_archive) $(olm_clusterserviceversion_path) | $(NEEDS_HE
 		$(YQ) - $(abspath $(olm_clusterserviceversion_path)) $(abspath $(olm_examples)) $(abspath $(olm_additional_manifests)) |\
 		$(OPERATOR-SDK) generate bundle --output-dir . --package $(olm_project_name) --version $(VERSION:v%=%)
 
+	@# Set the previous version in the replaces field, this is required for auto-upgrade to work
+	$(YQ) -i '.spec.replaces = "$(olm_project_name).$(olm_replaces_version)"' $(olm_bundle_dir)/manifests/$(olm_project_name).clusterserviceversion.yaml
```

Comment:
Seems like this should work, based on the documentation.
 * https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/#replaces

Maybe  `skipRange` would be worth adding in a future PR, to avoid having to upgrade through every patch version.